sdk/servicebus/azure-servicebus/azure/servicebus/_base_handler.py

```diff
@@ -285,13 +288,22 @@ def _do_retryable_operation(self, operation, timeout=None, **kwargs):
                 self._backoff(
                     retried_times=retried_times,
                     last_exception=last_exception,
-                    timeout=timeout
+                    abs_timeout_time=abs_timeout_time
                 )
 
-    def _mgmt_request_response(self, mgmt_operation, message, callback, keep_alive_associated_link=True, **kwargs):
-        # type: (str, uamqp.Message, Callable, bool, Any) -> uamqp.Message
+    def _mgmt_request_response(
+        self,
+        mgmt_operation,
+        message,
+        callback,
+        keep_alive_associated_link=True,
+        timeout=None,
```

Comment:
Should it be timeout_ms?

Should it be flexible enough to take a timedelta?

Is there precedent for this elsewhere we should be consistent to?

Reply:
re: time unit. This is a good naming suggestion for being clear. I believe there's a pattern: eh&sb respect timeout in unit of seconds while uamqp respects units of milliseconds, and we finally pass *1000 to the final uamqp call. But it would be good to have explicitness here, will apply the change.

re: timedelta, I don't see azure-core/guideline mentioned that we take `timedela` as timeout. In azure-core timeout is type of integer and I don't see "timedelta" being taken as timeout in other sdks (e.g. storage, cosmos).

Reply:
okay, I've given up the idea of renaming it to "timeout_seconds" cuz it would break the internal retry logic which now relies on the parameter name `timeout`(`kwargs.pop("timeout",None)`).

instead, I've added some comments to the method hope it would be clear enough :)